manifests/3.0.0/opensearch-dashboards-3.0.0.yml

```diff
@@ -55,6 +55,3 @@ components:
   - name: alertingDashboards
     repository: https://github.com/opensearch-project/alerting-dashboards-plugin.git
     ref: main
-  - name: securityAnalyticsDashboards
-    repository: https://github.com/opensearch-project/security-analytics-dashboards-plugin.git
```

Comment:
Suggest commit it out instead of removing in case we forgot, you can add a line of comment about why.

Reply:
Done!